src/utils/mockPortals.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable func-names */
+/* eslint-disable react/no-find-dom-node */
+
+// Dirty work-around until Enzyme fully supports React Portals rendered with shallow()
+// See: https://github.com/airbnb/enzyme/issues/1507
+// Idea for this taken from:
+// https://github.com/mui-org/material-ui/commit/aa10940b584239dce62ae5fd1d6c379ddf66b663
+
+// mockPortal.init() mocks the render() function of Paragon's Modal so that it renders directly
+// without a Portal.
+// mockPortal.reset() returns the render() function back to normal.
+
+import ReactDOM from 'react-dom';
+import { Modal } from '@edx/paragon';
+
+const portalOrigin = {};
+
+const mockPortal = {
+  init: () => {
+    portalOrigin.render = Modal.prototype.render;
+    Modal.prototype.render = function () {
+      return this.renderModal();
+    };
+
+    portalOrigin.getMountNode = Modal.prototype.getMountNode;
```

Comment:
What is getMountNode? I don't see getMountNode in the Modal.

Reply:
Oh oops. I copied this code from another project and I just assumed this was a React thing, but it's actually specific to that other project. I'll remove this.